plugins/meta/flannel/flannel.go

```diff
@@ -36,8 +36,16 @@ import (
 )
 
 const (
-	defaultSubnetFile = "/run/flannel/subnet.env"
-	defaultDataDir    = "/var/lib/cni/flannel"
+	defaultSubnetFile         = "/run/flannel/subnet.env"
+	defaultDataDir            = "/var/lib/cni/flannel"
+	delegateKeyType           = "type"
```

Comment:
I don't think it's necessary to define constants for all the struct keys.

Reply:
ok, thank you , and is there any other problem except this ?